gitlab/v4/objects.py

```diff
@@ -1764,6 +1780,65 @@ def create(self, data, **kwargs):
         return CreateMixin.create(self, data, path=path, **kwargs)
 
 
+class ProjectPipelineScheduleVariable(SaveMixin, ObjectDeleteMixin, RESTObject):
+    _id_attr = 'key'
+
+
+class ProjectPipelineScheduleVariableManager(CRUDMixin, RESTManager):
+    _path = '/projects/%(project_id)s/pipeline_schedules/%(pipeline_schedule_id)s/variables'
+    _obj_cls = ProjectPipelineScheduleVariable
+    _from_parent_attrs = {'project_id': 'project_id',
+                          'pipeline_schedule_id' : 'id'}
+    _create_attrs = (('pipeline_schedule_id', 'key', 'value'), tuple())
+    _create_attrs = (('key', 'value'), tuple())
+
+    def list(self):
```

Comment:
I'm not sure it's a good idea to implement this method. I'd like to avoid adding features that are not directly coming from the GitLab API. I'd rather remove this method.